src/racer/core.rs

```diff
@@ -997,9 +997,9 @@ fn complete_from_file_(
             let line = src_text[linestart..pos].trim().rsplit(';').nth(0).unwrap();
             debug!("Complete path with line: {:?}", line);
 
-            /// Test if the **path expression** starts with `::`, in which case the path
-            /// should be checked against the global namespace rather than the items currently
-            /// in scope.
+            // Test if the **path expression** starts with `::`, in which case the path
```

Comment:
Thanks for cleaning these up; it makes me sad to lose the ability to add doc comments to locals and have them show up in completions, but the rustdoc team has spoken.

Reply:
Did they actually consider that use case? It might be that they just missed it, and deprecated this as an useless thing.

Reply:
They considered it; I asked them about it, which was what prompted the discussion which ended in deprecation.